src/js/components/Users/UserList.js

```diff
@@ -0,0 +1,40 @@
+import { $ } from '../../utils/common.js';
+import { onUserHandler, rippleActiveHandler } from '../../utils/event.js';
+
+export default function UserList() {
+  this.$Users = [];
+  this.$UsersDom = $('#user-list');
+
+  this.setState = (users = []) => {
+    this.$Users = users;
+    this.render();
+  }
+
+  this.render = () => {
+    const users = Array.from(this.$Users).map(user => template(user));
+    this.$UsersDom.innerHTML += users.join('');
+
+    const userButtons = document.createElement('div');
+    userButtons.style.display = 'inline-block';
+    
+    const userCreateButton = document.createElement('button');
+    userCreateButton.classList.add('ripple', 'user-create-button');
+    userCreateButton.textContent = '+ 유저 생성';
+    userButtons.appendChild(userCreateButton);
+
+    const userDeleteButton = document.createElement('button');
+    userDeleteButton.classList.add('ripple', 'user-delete-button');
+    userDeleteButton.textContent = '삭제 -';
+    userButtons.appendChild(userDeleteButton);
+
+    this.$UsersDom.appendChild(userButtons);
+    this.$UsersDom.addEventListener('click', onUserHandler);
+    rippleActiveHandler(this.$UsersDom.firstChild);
+  }
+}
```

Comment:
버튼 생성은 반복되는 부분 같은데, 버튼을 생성하는 별도의 메소드를 만들어도 좋았을 것 같아요~! 

```suggestion
this.createButton = (classLists, textContent) => {
    const button = document.createElement('button');
    button.classList.add(...classLists);
    button.textContent = textContent;

    return button;
};

this.render = () => {
    const users = Array.from(this.$Users).map(user => template(user));
    this.$UsersDom.innerHTML += users.join('');

    const userButtons = document.createElement('div');
    userButtons.style.display = 'inline-block';
    
    const userCreateButton = this.createButton(['ripple', 'user-create-button'], '+ 유저 생성');
    userButtons.appendChild(userCreateButton);

    const userDeleteButton = this.createButton(['ripple', 'user-delete-button'], '삭제 -');
    userButtons.appendChild(userDeleteButton);

    this.$UsersDom.appendChild(userButtons);
    this.$UsersDom.addEventListener('click', onUserHandler);
    rippleActiveHandler(this.$UsersDom.firstChild);
  }
}
```